app/assets/javascripts/track.js.coffee

```diff
@@ -0,0 +1,30 @@
+App.Track =
+  
+  initialize: ->
+
+    if _paq?
+
+      # Custom Variables
+      $('html[data-js-usertype]').each ->
+        $this = $(this)
+        $usertype = $this.data('js-usertype')
```

Comment:
Por convención, las variables en coffeescript y javascript que llevan un dolar delante son aquellas que "apuntan" a nodos del dom con jquery. La variable `$this` cumple esta convención (está apuntando a un nodo del dom, en este caso el html raíz), pero la variable `$usertype` no (es una cadena de texto). Así que esta última (y las otras de este fichero en la misma situación) no deberían llevar el $.